config/scout.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Search Engine
+    |--------------------------------------------------------------------------
+    |
+    | This option controls the default search connection that gets used while
+    | using Laravel Scout. This connection is used when syncing all models
+    | to the search service. You should adjust this based on your needs.
+    |
+    | Supported: "algolia", "null"
+    |
+    */
+
+    'driver' => env('SCOUT_DRIVER', 'algolia'),
```

Comment:
Should we default to `null` so this can be deployed without `ALGOLIA_APP_ID` and `ALGOLIA_SECRET` being set? We could then set `SCOUT_DRIVER=algolia` on each environment as a sort of feature flag?

Reply:
I think it's technically fine since I'm not actually using the `Searchable` trait in a model yet, so none of this config is technically being used, but that being said, I do think this suggestion makes sesne!